AmericanMedicalAssociation/AMA-style-guide
# EWL-4227 Update handlebars

## Ticket(s)

Please do not submit a Pull Request without a relevant ticket! If you are creating a new pattern or feature, create an issue describing the need for this feature Github **first** and then link to it below.

**Github Issue**

- [Issue XXX: Issue Title](https://github.com/AmericanMedicalAssociation/AMA-style-guide/issues/XXX)

**Jira Ticket**

- [EWL-4277: Update Handlebars](https://issues.ama-assn.org/browse/EWL-4277)


## Description

In order to update the outdated dependencies I've had to fork both https://github.com/AmericanMedicalAssociation/gulpicon and https://github.com/AmericanMedicalAssociation/grunticon-lib. These libraries haven't been updated in over a year, but we should still probably periodically check on them to pull updates if they ever happen. 

## To Test

- `npm install`
- `gulp serve`
- observe icons still work
- search through `package-lock.json` and observe no instances of handlebars < 4

## Relevant Screenshots/GIFs

Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.


## Remaining Tasks

Remaining tasks?


## Additional Notes

Anything more to add?
